server.py

```
@@ -55,23 +55,366 @@
+    # not the wizard's finish_setup, so the LaunchAgent plist has
+    # to be installed/removed here too.
+    as_msg = _sync_autostart_with_config(config)
+    if as_msg:
+        messages.append(as_msg)
+
+    ok, reload_msg = _reload_langgraph_agent()
+    messages.append(reload_msg)
+    return ok, messages
+
+
+def _build_settings_callbacks() -> dict[str, Any]:
```

Ruff (0.15.10)

[warning] Too many statements (56 > 50)

(PLR0915)